### PR TITLE
Many dependencies need to be bumped

### PR DESCRIPTION
**Pull request by i386**
# Description

Bump it all

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -368,12 +369,12 @@
...
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-branch-source</artifactId>
-            <version>2.2.3</version>
+            <version>2.3.2</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-multibranch</artifactId>
-            <version>2.16</version>
+            <version>${workflow.support.version}</version>
```
Comment (kzantow) [inline pom.xml]: Is this always going to be the same as `workflow-support`?

Comment (vivek) [reply pom.xml]: Yeah this doesn't seem right. They are different plugins and can have different versions.

Comment (i386) [reply pom.xml]: That was my IDE being shit. Sorry about that.

**Comment by michaelneale** [comment]
wow the previous failure - NOTHING worked in the ATH (probably means it didn't load).

**Comment by michaelneale** [comment]
won't some of these plugins need newer version of LTS than blue ocean or ATH currently supports?

File: pom.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
   <properties>
     <java.level>8</java.level>
-    <jackson.version>2.8.9</jackson.version>
+    <jackson.version>2.8.10</jackson.version>
     <jenkins.version>2.73.3</jenkins.version> <!-- Should be kept in sync with Dockerfile FROM statement-->
```
Comment (michaelneale) [inline pom.xml]: this and the ATH version in the Jenkinsfile *may* need to be bumped to latest LTS if some of these new deps require it

**Comment by michaelneale** [comment]
:bug: need to try the ATH - it likely isn't loading blue ocean at all.

**Comment by i386** [comment]
@michaelneale latest `workflow-job` needs the latest LTS. Ive downgraded.

**Comment by i386** [comment]
Legit test failure resulted in [JENKINS-48644](https://issues.jenkins-ci.org/browse/JENKINS-48644) which is being tracked down by @svanoort

**Comment by michaelneale** [comment]
nice - the system works! ish.

**Comment by vivek** [comment]
Closing, not relevant anymore.

**Pull request closed without merging**